examples/haas.cfg

```diff
@@ -114,3 +114,7 @@ haas.ext.auth.database =
 # are required.
 vlans = 100-200
 
+[haas.ext.switches.dell]
+# Make modifications to the dell switches' configuration persistent
```

Comment:
Please start the sentence with something like `# Uncomment to make modifications...`